gradle-conjure/src/main/java/com/palantir/gradle/conjure/CompileConjureTypeScriptTask.java

```diff
@@ -38,10 +40,14 @@ public final void setProductDependencyFile(File productDependencyFile) {
     }
 
     public CompileConjureTypeScriptTask() {
-        doFirst(task -> {
```

Comment:
:+1:

```
 Task :log-receiver-api:compileConjureTypeScript
Caching disabled for task ':log-receiver-api:compileConjureTypeScript' because:
  Build cache is disabled
Task ':log-receiver-api:compileConjureTypeScript' is not up-to-date because:
  Additional action for task ':log-receiver-api:compileConjureTypeScript': was implemented by the Java lambda 'com.palantir.internalpublish.conjure.PublishConjureTypeScriptPlugin$$Lambda$1026/0x0000000800c97c40'. Using Java lambdas is not supported, use an (anonymous) inner class instead.
```